.travis.yml

```diff
@@ -1,12 +1,44 @@
+sudo: false
+dist: trusty
+
 language: php
+
 php:
+  - '5.6'
   - '7.0'
   - '7.1'
   - '7.2'
   - nightly
 
+git:
+  depth: 1
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: '5.6'
+    - php: nightly
+
+cache:
+  apt: true
+  timeout: 604800
```

Comment:
I'm under the impression that setting the timeout here likely isn't necessary. The build is fast enough and I'd prefer to not have to debug issues around a stale cache.